src/main/java/org/jqassistant/contrib/plugin/php/scanner/parser/Helper.java

```diff
@@ -245,19 +254,30 @@ private String function_getFullQualifiedName(String Name, PHPNamespaceDescriptor
         return "FUNCTION|" + fullname;
     }
     
+    /**
+     * create new property
+     * @return PHPPropertyDescriptor
+     */
     public PHPPropertyDescriptor getProperty(){
```

Comment:
I think there are better terms than `get` for creating a new descriptor.

Same goes for the other functions as well.